bot/exts/evergreen/trivia_quiz/_constants.py

```diff
@@ -0,0 +1,30 @@
+QUESTION_LIMIT = 4
+HINTS_LIMIT = 2
+CORRECTION_FUZZ_RATIO = 85
+MESSAGE_TIMEOUT = 15
+
+
+WRONG_ANS_RESPONSE = (
+    "No one answered correctly!", "Better luck next time"
+)
+WINNER_RESPONSE = (
+    "Congratulations {winners_mention} :tada: "
+    "{winner_word} have won this quiz game with a grand total of {highest_points} points!"
+)
```

Comment:
```suggestion
WINNER_RESPONSE = (
    "{winners_mention} Congratulations "
    "on winning this quiz game with a grand total of {highest_points} points :tada:"
)
```
This will also get rid of the `word` variable in _helpers.py